packages/block-library/src/block/edit.js

```diff
@@ -134,6 +135,7 @@ function getContentValuesFromInnerBlocks( blocks, defaultValues ) {
 	/** @type {Record<string, { values: Record<string, unknown>}>} */
 	const content = {};
 	for ( const block of blocks ) {
+		if ( block.name === blockName ) continue;
```

Comment:
I think this check could be clearer to the reader (I had to scroll up to find where `blockName` comes from to understand what value it has).

Perhaps calling it `patternBlockName` or adding a comment above this code could help.